pkg/serviceaccounts/controllers/docker_registry_service.go

```diff
@@ -48,7 +48,7 @@ type serviceLocation struct {
 
 var serviceLocations = []serviceLocation{
 	{namespace: "default", name: "docker-registry"},
-	{namespace: "openshift-image-registry", name: "registry"},
+	{namespace: "openshift-image-registry", name: "image-registry"},
```

Comment:
My pull adding this says "Makes the controller bilingual for both supported locations."

Are you sure you don't need to keep this?

Reply:
`docker-registry.default` is the "other supported location"

"registry.openshift-image-registry" is/was never a supported location